cli/remote_build.go

```diff
@@ -451,45 +393,6 @@ func fetchProject(urls []string) (*Project, error) {
 	return &project, nil
 }
 
-func fetchUserEmail() (string, error) {
```

Comment:
Another thing I was wondering. I didn't find where it was being used. Do we really won't use such information even in the near future?